### DISCUSSION TITLE
[Nova3.0]Resource Field Report

### DISCUSSION THREAD

**Discussion by ss071109** [laravel/nova-issues#3601] [Q&A] [answered]
- Laravel Version: 8.0
- Nova Version: 3.0
- PHP Version: 7.3

### Description:
Resource{
order,
schoolUser,
schoolClass
}
When create a field use "Text::make('class', 'schoolUser.schoolClass.class_name')" is work.

But I go to order resource index sort 'class' is not working.

### Steps To Reproduce
1. Use code like "Text::make('class', 'schoolUser.schoolClass.class_name')" create field.
2. Click sort button in resource index

### Screenshots:
<img width="1229" alt="螢幕截圖 2021-10-22 下午8 37 05" src="https://user-images.githubusercontent.com/92988901/138454815-f64b6203-5803-448a-b129-2017d0078147.png">

**Comment by crynobone** [comment, marked as answer]
This operation is simply not supported
